src/components/pages/search-page/SearchPage.css

```
@@ -0,0 +1,50 @@
+.search-page-form {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-row-gap: 20px;
+  justify-items: center;
+  align-items: center;
+}
+
+.search-page-form-field {
+  margin: 0;
+}
+
+/* .search-page-form-field :first-child {
+  margin-left: 0;
+} */
+
+.search-page-form-field-button {
+  width: 50%;
+  height: 50%;
+  border-radius: 5px;
+  margin: 20px 0px;
+}
+
+@media (max-width: 980px) {
+  .search-page-form-field {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+  }
+  /* .search-page-form-field :first-child {
+    width: 90%;
+  } */
+}
+
+@media (max-width: 640px) {
+  .search-page-form {
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: repeat(6, 1fr);
+  }
+
+  .search-page-form-field {
+    width: 70%;
```

Если здесь указать такую ширину, то блок снизу можно удалить. А этот нижний блок лучше убрать, он слишком неявный.

```suggestion
    width: 166px;
```